physlr/physlr.py

```diff
@@ -998,10 +1008,26 @@ def physlr_subgraphs(self):
             os.makedirs(self.args.output)
         for u in progress(vertices):
             vertices_u = set()
+            u_mol = set()
+            if u in g:
+                u_mol.add(u)
+            else:
+                mol = 0
+                to_mol = u + "_" + str(mol)
```

Comment:
Solution looks good to me -- just double checking that it wouldn't be possible for there to be a molecule ID skipped in the input you are using? (ie. A_1, A_3 but no A_2).

Reply:
Um, I am not 100% sure, theoretically this case is not possible as we sort the communities by their size and then make molecules. and at the end we remove singletons. I believe singletons must be last instances. But will check the files to see if I can find such a case.

Reply:
I just checked a `.mol.tsv` file of `10X Chromium` which has higher barcode multiplicity. checking 100000 cases I found no occurrence of a skipped ID.
```
[aafshinfard@hpce704 data]$ head /projects/btl/jowong/github/physlr/data_hg004_ntHits/bf_repetitive/threshold90/hg004.k40-w32.n100-5000.c2-x.physlr.overlap.n67.k3.mol.tsv -n100000 > ./head.tsv
[aafshinfard@hpce704 data]$ cat head.tsv | tail -n999999 > head2.tsv
[aafshinfard@hpce704 data]$ cat head2.tsv | awk '{print $1}' > head.tsv
[aafshinfard@hpce704 data]$ head head.tsv | cut -d'_' -f2 | awk '($1 > prev+1) { print prev } { prev = $1 }'
```